### PR TITLE
feat: added 'help' button on header

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* feat: added 'help' button on header
  * removed 'Connection Trouble' button from sidebar
* fix: updated header
  *  removed 'Network' text from network button
  *  displays address even more shorter  
* fix: display 'EVM deposit' row on the native account box only if users have balance on EVM deposit
* fix: removed release notes from PULL_REQUEST_TEMPLATE

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**This pull request makes the following changes:**

**Changes**

=Before=
image

=After=
image

### CODE REVIEW

**Review by sirius651 — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -11,11 +11,6 @@
 - [ ] documentation changes
 - [ ] tested on mobile devices
 
-**Release notes:**
```
Comment (bobo-k2) [inline .github/PULL_REQUEST_TEMPLATE.md:14]: Why do you removed this section? I thought that we generate release note document from this

**Review by bobo-k2 — APPROVED** [review]

**Comment by Kahonnohak** [comment]
Very nice thanks, 

Question, I rejected on this screen and storage cleared but I cannot authorise anymore. nothing comes up and no warning on the extension... Do you know what can be done? There is nothing on polkadot.js app too
Screenshot 2023-05-09 at 14 08 18

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (impelcrypto) [reply .github/PULL_REQUEST_TEMPLATE.md:14]: > I thought that we generate release note document from this
It looks the title of PR is used for generating release note. @niklabh Please correct me if I'm wrong.

**Comment by impelcrypto** [comment]
@Kahonnohak Users have to configure it on "Manage Website Access" manually. Maybe you can add a tip on 'Connection Troubles' modal later 
image

**Review by Kahonnohak — APPROVED** [review]
LGTM

**Comment by Kahonnohak** [comment]
Great work!

**Pull request merged**